convert2rhel/backup/certs.py

```diff
@@ -187,7 +187,8 @@ def _restore(self):
                 # Resolves RHSM error when removing certs, as the system might not have installed any certs yet
                 loggerinst.info("No certificates found to be removed.")
             else:
-                loggerinst.error("OSError({0}): {1}".format(err.errno, err.strerror))
+                # Will be handled in BackupController
+                raise err
```

Comment:
Can remove the else here. Codecov also claims this isn't tested

Reply:
I think I can't remove the else path there. Because above, there is a test if [`err.errno == errno.ENOENT`](https://github.com/oamg/convert2rhel/blob/main/convert2rhel/backup/certs.py#L186) and if yes, info is just printed about there is nothing to be removed and in that case is no need to raise the exception.

So I'll have to add the unit tests for this line.

Reply:
You could just: 

```python
except OSError as err:
            if err.errno == errno.ENOENT:
                # Resolves RHSM error when removing certs, as the system might not have installed any certs yet
                loggerinst.info("No certificates found to be removed.")
                return
                
          loggerinst.error("OSError({0}): {1}".format(err.errno, err.strerror))
          # Will be handled in BackupController
          raise err
```

Reply:
Except with the crappy formatting, but you could return in that if statement.

Reply:
Oh! Right, thanks :)